test/test-golden/v2020-12-remote.md

```diff
@@ -71,8 +71,55 @@ An array of three fractional numbers.
     * `[1.3, 4.03, 42]`
     * `[18, 0.1, 1.1]`
 
+### Image.metadata
+
+* **Type**: `object`
+* **Required**: No
+* **Additional properties are allowed.**
+* **Type of each property**: [`meta`](#reference-meta)
```

Comment:
Demonstration that the additional property type properly creates a link now (though I tried and failed to get it to use the title instead of the type).